common/cards/default/effects/bed.ts

```diff
@@ -18,10 +18,10 @@ class BedEffectCard extends EffectCard {
 	}
 	override canAttach(game: GameModel, pos: CardPosModel) {
 		const result = super.canAttach(game, pos)
-		const {currentPlayer} = game
```

Comment:
This results in bed being able to be placed on the opponent's active hermit - is this intentional?

Reply:
`EffectCard.canAttach` already checks for trying to attach to opponent, this should make checks for Emerald pass even if players have different active row indexes

Reply:
Nice